src/app/helpers/logger.js

```diff
@@ -0,0 +1,63 @@
+// Node.js logger utility using winston
+const fs = require('fs');
+const path = require('path');
+const { createLogger, format, transports } = require('winston');
+
+const { combine, label, printf, simple, timestamp } = format;
+
+const LOG_LEVEL = 'debug';
+const LOG_FILE = 'app.log';
+let LOG_DIR = 'log';
+
+console.log(process.env); // eslint-disable-line no-console
+
+LOG_DIR = process.env.SIMORGH_LOG_DIR || LOG_DIR;
```

Comment:
```suggestion
const localLogPath = 'log';
const LOG_DIR = process.env.SIMORGH_LOG_DIR || localLogPath;
```

This feels more explicit to me